src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/MsQuicEnums.cs

```diff
@@ -69,10 +69,10 @@ internal enum QUIC_STREAM_OPEN_FLAGS : uint
     internal enum QUIC_STREAM_START_FLAGS : uint
     {
         NONE = 0x0000,
-        FAIL_BLOCKED = 0x0001, // Only opens the stream if flow control allows.
```

Comment:
We should look into using MsQuic's [generated interop](https://github.com/microsoft/msquic/blob/main/src/cs/lib/msquic_generated.cs) layer instead of doing all this manually in .NET. Several of these enums are out of date, and don't include all the latest values.

Reply:
Sounds good!